eregs_core/management/commands/import_diff.py

```diff
@@ -145,5 +149,6 @@ def recursive_insert(node, version):
                 recursive_insert(child, version)
 
     recursive_insert(node, version)
-    #DiffNode.objects.bulk_create(result_nodes)
+
+    RegNode.objects.bulk_create(result_nodes)
```

Comment:
Could you add a [`batch_size`](https://docs.djangoproject.com/en/1.8/ref/models/querysets/#django.db.models.query.QuerySet.bulk_create) here? I had to do this [here](https://github.com/cfpb/eregs-2.0/pull/39/files#diff-4a603ca10ec94b4f2c88fe2495d031b2R130) because with some large files the size of the data being passed in a single query exceeded the DB limit.